ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -414,9 +414,11 @@ protected int determineActiveToolState() {
     }
 
     private int determineLanguageState(final int languageIndex) {
-        if (mManifests.get(languageIndex) != null) {
-            return STATE_LOADED;
-        } else if (isSyncToolsDone() && mTranslations.indexOfKey(languageIndex) >= 0 &&
+        final Manifest manifest = mManifests.get(languageIndex);
+        final boolean syncDone = isSyncToolsDone();
+        if (manifest != null && syncDone) {
+            return Manifest.Type.TRACT == manifest.getType() ? STATE_LOADED : STATE_NOT_FOUND;
+        } else if (syncDone && mTranslations.indexOfKey(languageIndex) >= 0 &&
```

Comment:
I'm definitely going to want to re-examine this. if we have a manifest we want to show it even if there is still a sync process running.

Reply:
I can remove it I wasn't sure if we should be displaying the screen if it was still syncing.